spec/controllers/admin/base_controller_spec.rb

```diff
@@ -9,9 +9,6 @@ def index
 
   let(:account) { create :account }
 
-  before do
-    allow(controller).to receive(:setup_request)
```

Comment:
this broke down with some gem update, rails maybe?, but seems not necessary anyway